src/Spice86.Core/Emulator/InterruptHandlers/Bios/SystemBiosInt15Handler.cs

```diff
@@ -20,14 +23,25 @@ public class SystemBiosInt15Handler : InterruptHandler {
     }
 
     /// <summary>
-    /// No extended memory size present. Yet.
-    /// Reports 0 in AX.
+    /// Reports extended memory size in AX.
     /// </summary>
     public void GetExtendedMemorySize() {
-        //We've got no extended memory (yet)
         _state.AX = 0;
     }
 
+    public void CopyExtendedMemory(bool calledFromVm) {
+        bool enabled = _memory.IsA20Enabled;
+        _machine.Memory.EnableOrDisableA20Gate(true);
+        uint bytes = (uint)(_state.CX * 2);
+        uint data = MemoryUtils.ToPhysicalAddress(_state.ES, _state.SI);
+        long source = _memory.UInt32[data + 0x12] & 0x00FFFFFF + _memory.UInt8[data + 0x16] << 24;
```

Comment:
Looks like a struct in memory. Maybe having a full class would make the intent a bit clearer?

Reply:
Code removed.